### ISSUE TITLE
Use "branches: master" instead of "skip_branches" when you want to just use master in job configs

### ISSUE THREAD

**Issue by imkin** [kubernetes/test-infra#11968]
I see this across various job configs (Example: 
[config/jobs/kubernetes/sig-gcp/gpu/sig-gcp-gpu-presubmit.yaml#L14)](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-gcp/gpu/sig-gcp-gpu-presubmit.yaml#L14)

The intention here seems to be to just run the job for the "master" branch. We already have a way to do this by using "branches" (as we see this for the similar config here [config/jobs/kubernetes/sig-gcp/gpu/sig-gcp-gpu-presubmit.yaml#L105](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-gcp/gpu/sig-gcp-gpu-presubmit.yaml#L105))
I recommend just doing "branches: master" everywhere we see this pattern.

/cc @amwat @cjwagner @krzyzacy @spiffxp

**Comment by BenTheElder** [comment]
-1, [sig-testing slack thread](https://kubernetes.slack.com/archives/C09QZ4DQB/p1553715011860700)

TLDR feature branch testing relies on this being a skip.

**Comment by imkin** [comment]
This seems to be a way of indicating "default" jobs that need to run on all branches "except" the released ones.
